src/plone/api/content.py

```diff
@@ -187,7 +187,7 @@ def move(source=None, target=None, id=None, safe_id=False):
     # If no target is given the object is probably renamed
     if target and source.aq_parent is not target:
             target.manage_pasteObjects(
-                source.aq_parent.manage_cutObjects(source_id)
+                source.aq_parent.manage_cutObjects(source_id),
```

Comment:
I tend to agree with @jensens: enforcing trailing commas all over the places makes no sense and is against out own code style guidelines; this should be written instead:

```python
target.manage_pasteObjects(
    source.aq_parent.manage_cutObjects(source_id))
```